### PR TITLE
Enable more rubocop tests

### PR DESCRIPTION
**Pull request by bjk-soundcloud**
Enable additional cops:
* Style/ParenthesesAroundCondition
* Style/SignalException
  - EnforcedStyle: raise
* Style/AlignParameters
* Style/Blocks
* Style/RedundantReturn
* Style/RedundantSelf
* Style/AccessModifierIndentation
* Style/IndentationWidth
* Style/IndentationConsistency
* Style/NegatedIf

### CODE REVIEW

**Comment by ddfreyne** [comment]
:+1:

The README can also be updated to remove Ruby 1.8.x-style hashes.

**Comment by arthurnn** [comment]
:+1: lemme know when this is done, so I can have a final look.
thanks

**Comment by bjk-soundcloud** [comment]
@arthurnn Ok, ready for review.  There are a couple of cops that I tried to enable, but caused test failures.  This is an easier working list.

File: lib/lhm.rb [hunk 1]
```diff
@@ -96,7 +96,7 @@ def setup(adapter)
   def adapter
     @@adapter ||=
       begin
-        raise 'Please call Lhm.setup' unless defined?(ActiveRecord)
+        fail 'Please call Lhm.setup' unless defined?(ActiveRecord)
```
Comment (arthurnn) [inline lib/lhm.rb]: I hate this `fail` instead of `raise` on rubocop, I think this should be one we ignore!

**Comment by bjk-soundcloud** [comment]
@arthurnn Ok, I fixed it so Style/SignalException will use `raise`

**Comment by arthurnn** [comment]
thanks a lot @bjk-soundcloud

**Pull request merged**